src/js/themes/base.js

```diff
@@ -524,7 +524,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       label: {
         margin: { vertical: 'xsmall', horizontal: 'small' },
       },
-      margin: { bottom: 'small' },
+      margin: {
```

Comment:
This is a breaking change. Why do we need to change the default margin in the base theme?